CustomTextAnalytics.CognitiveSearch/Solution/IndexingCliTool/CognitiveSearchIntegration.Core/Services/Indexers/CustomTextIndexingService.cs

```diff
@@ -62,6 +62,12 @@ public SearchIndex CreateIndex(CustomTextSchema schema, string indexName)
                 IsKey = true
             });
 
+            // document name
+            indexFields.Add(new SearchField("document_name", SearchFieldDataType.String));
+
+            // document URI
+            indexFields.Add(new SearchField("document_uri", SearchFieldDataType.String));
```

Comment:
did you test out clicking on the link?
i recall this wasn't working!

Reply:
Yes I did and it worked for me.